features/upgrade/machines/machines.feature

```diff
@@ -75,3 +75,62 @@ Feature: Machine-api components upgrade tests
     When I scale the machineset to -1
     Then the step should succeed
     And the machineset should have expected number of running machines
+
+  @upgrade-prepare
+  @admin
+  Scenario: Cluster should automatically scale up and scale down with clusterautoscaler deployed - prepare
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    And I use the "openshift-machine-api" project
+    When I run the :create admin command with:
+      | f | <%= BushSlicer::HOME %>/testdata/cloud/cluster-autoscaler.yml |
```

Comment:
please use step
```
I obtain test data file "cloud/cluster-autoscaler.yml"
```
then use the file.

Reply:
and all other places where `BushSlicer::HOME`

Reply:
Updated, thanks